tools/pipelines-tasks/azure-pipelines/build-vsix.yml

```diff
@@ -17,8 +17,8 @@ variables:
   tasksRoot: 'tools/pipelines-tasks'
   buildOutRoot: '$(Build.ArtifactStagingDirectory)/buildOutput'
   # Version number
-  major: '1'
-  minor: '1'
+  major: '2'
+  minor: '0'
```

Comment:
As it involved addition of new task, I preferred to make it version 2.0.0. Probably any additional features in existing task can go as minor version bump ?

Reply:
See https://semver.org

I believe adding a new task is backwards compatible. People using the existing tasks won't need to take any action because of the new task. It would be backwards incompatible if you were to change the inputs or behavior to the existing tasks in a way that would break customers, but that is not what you're doing.

Reply:
Got it, changed to minor version update